src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -58,11 +59,15 @@ public static ConversationStore getTestInstance(PersistentStorageAgent persisten
 
   /** The in-memory list of Conversations. */
   private List<Conversation> conversations;
+  private HashMap<UUID, Conversation> conversationMap_IdKeyValues;
```

Comment:
1) If you want to keep the variable name, you should still to the lowerCamelCase naming convention e.g., conversationmapIdKeyValues.

2) But I would suggest simply DROP the '_IdKeyValues'. This is easily inferred by the fact that the variable is a HashMap. Sometime we do NOT even say Map (i.e., we normally do NOT put the type of the variable inside the variable names). But you already have a 'conversations' variable, so 'conversationMap' to distinguish from that is okay.